build.cmd

```diff
@@ -67,7 +73,7 @@ call :build %*
 goto :AfterBuild
 
 :build
-%_buildprefix% msbuild "%_buildproj%" /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%_buildlog%";Append %* %_buildpostfix%
+%_buildprefix% msbuild "%_buildproj%" %defaultBuildConfig% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%_buildlog%";Append %* %_buildpostfix%
```

Comment:
Should adding %defaultBuildConfig% be conditioned on if /p:Configuration= isn't already otherwise specified? 

Or... if we do `msbuild /p:Configuration=Configuration1 /p:Configuration=Configuration2`, which one wins?